GentApp/ViewModels/UserViewModel.cs

```diff
@@ -53,11 +53,14 @@ public class UserViewModel : ViewModelBase {
 			}
 		}
 
-		public async void SaveUser() {
+		public async void SaveUser(string name) {
 			try {
```

Comment:
.

Reply:
Nee das om in te stellen wa dat u bericht toont als je iets opslaat. Bvb Branch saved ipv user saved

Reply:
hmmm